planner/core/plan_test.go

```diff
@@ -589,3 +590,51 @@ func (s *testPlanNormalize) BenchmarkEncodePlan(c *C) {
 		core.EncodePlan(p)
 	}
 }
+
+// Close issue 25729
+func (s *testPlanNormalize) TestIssue25729(c *C) {
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.Experimental.AllowsExpressionIndex = true
+	})
```

Comment:
This config is set to true permanently for the following tests. Do we need to restore it to the original value?

Reply:
Done in the new commit